en/peripherals/frsky_telemetry.md

```diff
@@ -35,16 +76,42 @@ You can see:
 Once you have the hardware connected, follow the instructions on
 http://ilihack.github.io/LuaPilot_Taranis_Telemetry/, Section **Taranis Setup OpenTX**.
 
-If you open the `LuaPil.lua` script with a text editor, you can edit the
-configuration. Suggested modifications:
+If you open the `LuaPil.lua` script with a text editor, you can edit the configuration. 
+Suggested modifications include:
 ```
 local BattLevelmAh = -1  -- use the battery level calculation from the vehicle
 local SayFlightMode = 0  -- there are no WAV files for the PX4 flight modes
 ```
 
+## FrSky Telemetry Receivers

```

Comment:
@bkueng I added section table as shown below. I think I've captured that these should work, but that the X series ones are the ones that we _know_ work. 

I changed the table telemetry input name to D.Port from HubProtocol. Is there anything that should be added or that is missing from the table (ie is it correct and useful?)

Reply:
The D4R-II works for sure as well.
The table looks good.